### ISSUE TITLE
MyQ Garage responds to On/Off

### ISSUE THREAD

**Issue by Riveloper** [NorthernMan54/homebridge-alexa#468]
Connected successfully to MyQ Liftmaster/Chamberlain garage doors. 

Devices show under Alexa as Garage Doors. 

Voice commands only work by saying:
"Turn door on" and "Turn door off" instead of "Open door" and "Close door".

**Comment by NorthernMan54** [comment]
Create a routine in Alexa with the wording you want.  This is a limitation on the Amazon side.

**Comment by rfiorentino1** [comment]
I'm also having this issue, but only on SOME devices. I set up myQ/homebridge at my house, in Tenessee, and this plugin, and everything works beautifully. I can say "open the garage" etc and it asks me for my code and all is well.

But I just got my parents a MyQ for christmas, and did the same thing, set up homebridge, etc, and their Alexa will only turn on/off, does not implement the voice code, and either says the garage door isn't responding, or doesn't support that if I ask it's status. Is there something I've done wrong, or something to do with the region stuff with Amazon cloud you were talking about in a previous issue? My parents are in New Jersey, which is why I thought of this, though that's really bad on Amazon's part. Thoughts/suggestions?
